packages/clerk-js/src/ui/components/OrganizationProfile/InviteMembersForm.tsx

```diff
@@ -99,6 +99,9 @@ export const InviteMembersForm = (props: InviteMembersFormProps) => {
               }),
             ),
           );
+        } else if (isClerkAPIResponseError(err) && err.errors?.[0]?.code === 'form_param_format_invalid') {
+          const message = err.errors[0]?.longMessage;
+          message && card.setError(message);
```

Comment:
@octoper shouldn't we make a localized message ?

Reply:
Agreed! We should use the `translateError` utility that `useLocalizations` provide, and add a translated string on the already existing `form_param_format_invalid` key.

Reply:
Good suggestion, just pushed the change, thank you @panteliselef and @anagstef!